src/cms/models/regions/region.py

```diff
@@ -158,6 +158,15 @@ class Region(models.Model):
         ),
     )
 
+    administrative_division_included = models.BooleanField(
+        default=False,
+        verbose_name=_("include administrative division into name"),
```

Comment:
```suggestion
        verbose_name=_("prepend administrative division to name"),
```
I think this would make it more clear the the administrative division goes before the name...
(German translation: "Verwaltungseinheit dem Namen voranstellen")

Reply:
Thank you for your suggestion, but actually I'm fine with include, so I want to keep it